admin/client/Forms.tsx

```diff
@@ -157,13 +157,40 @@ export interface NumberFieldProps {
     helpText?: string
 }
 
-export class NumberField extends React.Component<NumberFieldProps> {
+/**
+ * Like NumberField, but also capture the users text input
+ */
+export class NumberInputField extends React.Component<
+    NumberFieldProps<NumericInputValue>
+> {
+    render() {
+        const { props } = this
+
+        const textFieldProps = extend({}, props, {
+            value: props.value && props.value ? props.value.input : undefined,
```

Comment:
`props.value && props.value`? Is that a mistake?

Reply:
Yeah, that's a mistake. I believe I was intending to check props.value.input but there is no need. Thanks - will fix.